tox.ini

```diff
@@ -28,7 +25,14 @@ whitelist_externals =
                       /bin/sh
                       /bin/sed
 commands =
+         django18: pip-compile --output-file {envname}-requirements.txt tests/requirements.txt requirements/base.txt requirements/django18.txt
+         django19: pip-compile --output-file {envname}-requirements.txt tests/requirements.txt requirements/base.txt requirements/django19.txt
+         django110: pip-compile --output-file {envname}-requirements.txt tests/requirements.txt requirements/base.txt requirements/django110.txt
+         django111: pip-compile --output-file {envname}-requirements.txt tests/requirements.txt requirements/base.txt requirements/django111.txt
```

Comment:
I don't like the redundancy here, but I guess there is no way around it.

Reply:
I don't much like it either. How about this?

```diff
diff --git a/tox.ini b/tox.ini
index 05d246e77..efcfc2bb3 100644
--- a/tox.ini
+++ b/tox.ini
@@ -20,16 +20,16 @@ setenv =
     BUILDDIR = {envdir}
     CHROME_BIN = /usr/bin/google-chrome
     DJANGO_SETTINGS_MODULE = nav.django.settings
+    django18: DJANGOVER=18
+    django19: DJANGOVER=19
+    django110: DJANGOVER=110
+    django111: DJANGOVER=111
 passenv = WORKSPACE DISPLAY
 whitelist_externals =
                       /bin/sh
                       /bin/sed
 commands =
-         django18: pip-compile --output-file {envname}-requirements.txt tests/requirements.txt requirements/base.txt requirements/django18.txt
-         django19: pip-compile --output-file {envname}-requirements.txt tests/requirements.txt requirements/base.txt requirements/django19.txt
-         django110: pip-compile --output-file {envname}-requirements.txt tests/requirements.txt requirements/base.txt requirements/django110.txt
-         django111: pip-compile --output-file {envname}-requirements.txt tests/requirements.txt requirements/base.txt requirements/django111.txt
-
+         pip-compile --output-file {envname}-requirements.txt tests/requirements.txt requirements/base.txt requirements/django{env:DJANGOVER}.txt
          pip-sync {envname}-requirements.txt
          pip install -e .
```